examples/batch/v1beta1/jobqueue.yaml

```
@@ -0,0 +1,329 @@
+apiVersion: batch.aws.upbound.io/v1beta1
+kind: JobQueue
+metadata:
+  annotations:
+    meta.upbound.io/example-id: batch/v1beta1/jobqueue
+  labels:
+    testing.upbound.io/example-name: test_queue
+  name: test-queue
+spec:
+  forProvider:
+    computeEnvironmentOrder:
+      - computeEnvironmentSelector:
+          matchLabels:
+            testing.upbound.io/example-name: sample
+        order: 1
+    priority: 1
+    region: us-west-1
+    state: ENABLED
+---
+apiVersion: batch.aws.upbound.io/v1beta1
+kind: ComputeEnvironment
+metadata:
+  annotations:
+    meta.upbound.io/example-id: batch/v1beta1/computeenvironment
```

```suggestion
    meta.upbound.io/example-id: batch/v1beta1/jobqueue
```

And please fix the above example-ids.